components/crm/.env.example

```
@@ -1,20 +1,21 @@
 ENV_NAME=development
 
 # APP
-VERSION=v3.5.1
+VERSION=v3.6.0
 SERVER_PORT=4003
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVER_PORT key should go before the VERSION key

(UnorderedKey)